investigation-US-election-2016.Rmd

```diff
@@ -0,0 +1,254 @@
+---
+title: "investigation-US-election-2016"
+output: pdf_document
+---
+
+```{r knitropts, include = FALSE, echo = FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+# knitr::opts_chunk$set(tidy.opts = list(width.cutoff = 80), tidy = TRUE)
+```
+```{r libs, message = FALSE}
+library(conflicted)
+	conflicts_prefer(dplyr::filter, dplyr::lag)
+library(tidyverse)
+library(R6)
+library(formatR)
+```
+```{r readpolls, include = FALSE}
+polls = readr::read_delim("/home/mitch/courses/data-analysis/datasets/polls_us_election_2016.csv")
+```
+```{r usepolls}
+allstates = polls$state |>
+	unique() |>
+	sort()
+polls = polls |>
+	mutate(
+		interval = startdate %--% enddate,
+		middate = ymd(startdate + (enddate - startdate)/2),
+		.keep = "unused",
+		.after = "state"
+	) |>
+	mutate(
+		grade = grade |>
+			factor(levels = c("A+", "A", "A-", "B+", "B", "B-", "C+", "C", "C-", "D+", "D", "D-", NA)),
+		.keep = "unused",
+		.before = "samplesize"
+	)
+```
+```{r finalresults}
+finalresults = tibble(
+	state = allstates,
+	clinton = c(34.36, 36.55, 45.13, 33.65, 61.73, 48.16, 54.57, 53.09, 90.86, 47.82, 45.64, 62.22, 27.49, 55.83, 37.77, 41.74, 36.05, 32.68, 38.45, 47.83, 53.96, 40.98, 60.33, 60.01, 47.27, 46.44, 40.11, 38.14, 35.75, 33.70, 35.46, 44.92, 19.73, 47.92, 46.83, 55.45, 48.26, 59.01, 46.17, 27.23, 43.56, 28.93, 39.09, 47.46, 54.41, 40.67, 31.74, 34.72, 43.24, 48.18, 27.46, 56.68, 49.73, 52.54, 26.43, 46.45, 21.88),
```

Comment:
Why are you manually inserting the values when you downloaded them earlier

Reply:
I have a pdf containing them, but I'm not so good with parsing/scraping them. I figured I would be saving some time, doing it this way